src/event-stream/event-server.ts

```diff
@@ -190,6 +207,44 @@ async function handleClientMessage(msg: CoreNodeBlockMessage, db: DataStore): Pr
           value: hexToBuffer(event.contract_event.raw_value),
         };
         dbTx.contractLogEvents.push(entry);
+        if (
+          event.contract_event.topic === printTopic &&
+          event.contract_event.contract_identifier === bnsContractIdentifier
+        ) {
+          const functionName = getFunctionName(event.txid, parsedMsg.parsed_transactions);
+          if (nameFunctions.includes(functionName)) {
+            const attachment = parseNameRawValue(event.contract_event.raw_value);
+            const attachmentValue = await parseContentHash(attachment.attachment.hash);
+
+            const names: DbBNSName = {
+              name: attachment.attachment.metadata.name,
+              namespace_id: attachment.attachment.metadata.namespace,
+              address: addressToString(attachment.attachment.metadata.tx_sender),
+              expire_block: 0, // FIXME:
+              registered_at: parsedMsg.block_height,
+              zonefile_hash: attachment.attachment.hash,
+              zonefile: attachmentValue,
+              latest: true,
+              tx_id: event.txid,
+              status: attachment.attachment.metadata.op,
+              index_block_hash: parsedMsg.index_block_hash,
+              canonical: true,
+            };
+            console.log('update names ', JSON.stringify(names));
+            await db.updateNames(names);
+          } else if (functionName === namespaceReadyFunction) {
+            //event received for namespaces
+            const namespace: DbBNSNamespace | undefined = parseNamespaceRawValue(
+              event.contract_event.raw_value,
+              parsedMsg.block_height,
+              event.txid,
+              parsedMsg.index_block_hash
+            );
+            if (namespace != undefined) {
+              await db.updateNamespaces(namespace);
```

Comment:
Same thing here: should be `dbData.namespaces.push(namespace)`

Reply:
Updated this as suggested.